doc/data-warehouse-what-and-why.md

```diff
@@ -0,0 +1,258 @@
+satisfaction with a particular page.
+```
+
+These metrics are tracked daily.
+
+* Metrics about **content quality**:
+
+These metrics are tracked/updated on each new version.
+
+```
+Metrics that define core characteristics of content: the reading score, use of
+passive voice, guidelines compliance, number of paragraphs, words, complex sentences,...
+```
+
+## At scale and over time
+
+`At scale` and `over time` key principles for the Data Warehouse, because
+content designers are interested in:
+
+1. Metrics for each content item that is or have been live in GOV.UK (around
+   550K)
+2. Trends of each of the metrics **over time**
+
+To explain in detail why **over time** is important, let us explore three simple
+examples.
+
+### Example 1: Number of Content Items published by an organization
+
+Departments are content owners; this means that they are responsible for
+publishing, updating and archiving/retiring their content daily. Diligent
+management of content implies a steady pace of updates and publications while
+keeping the total number of content items in a sensible number.
+
+For example, if an organization is publishing at a very high rate for an
+extended period of time, it could be a signal that they need to improve the way
+they archive or retire existing content.
+
+To get these insights about content publication we can't focus on metrics for a
+single day, we need to understand how those metrics evolved through time; we
+need to explore metrics **over time**.
+
+Let's explore the example:
+
+* **Content Items published by an organization in one day**
+
+We have two organizations: `A` and `B`. In the last day, `Organisation A` has
+`550` live Content Items and `Organisation B` has `400`.
+
+With this information, it is not easy to say if any of those organization are
+not doing good management of their content because we only have a metric on a
+particular day of the year.
+
+![without metrics over time][figure-1-no-trends]
+
+* **Content Items published by an organization over time***
+
+If we track the number of content items **over time**, then we observe that
+`Organisation A` is decreasing the number of Content Items daily, and
+`Organisation B` has created too many content items in the last year.
+
+We can conclude that `organization b` would need to take action to improve the
+way they manage content over time: they can make an informed decision based on
+data about the way they manage content.
+
+![with metrics over time][figure-2-with-trends]
+
+### Example 2: Enforcing good practices
+
+Data is not only to help departments to manage their content over time; it can
+also help to provide evidence about best practices and guidelines compliance.
+
+For example, we discourage Departments from publishing content in PDF format.
+There are many reasons for it, but having some metrics to prove it would shorten
+the distance between the GOV.UK Guidelines and Department practices when
+publishing content.
```

Comment:
I think department and guidelines don't need to be capitalised. 

Not really sure what "shorten the distance" means?

Reply:
No no, the English is fine! Sorry, I should say I don't understand the point made by the whole sentence:

having some metrics to prove it would **shorten the distance** between the GOV.UK Guidelines and Department practices when publishing content

i.e what **_is_** shortening the distance GOV.UK Guidelines and Departments?

Reply:
I have updated it to:

```
For example, we discourage departments from publishing content in PDF format.
There are many reasons for it, but having some metrics to prove that PDFs are
not benefitial for users, would make it easier for departments to fully comply
with GOV.UK guidelines.
```